R/data_source_extension.R

```diff
@@ -2,11 +2,18 @@
 #' This was initially treated as an extension, but we had to include it in this package to make it run on the server,
 #' which effectively makes it not really an extension.
 #' @export
-get_riem_measures <- function(station = "SFO", date_start = "2020-01-01", date_end = NULL, full_columns = "Yes", tzone = ""){
+get_riem_measures <- function(station = "SFO", date_start = NULL, date_end = NULL, full_columns = "Yes", tzone = ""){
   loadNamespace("riem")
   if(is.null(date_end)){
     date_end <- as.character(Sys.Date())
   }
+  if (date_start == "" || is.null(date_start)) {
```

Comment:
Add some comment about -3.

Reply:
Should check null first.

Reply:
Added a comment and move the null check first.